src/garage/torch/value_functions/__init__.py

```diff
@@ -1,6 +1,6 @@
 """Value functions which use PyTorch."""
+from garage.torch.value_functions.gaussian_mlp_value_function import \
```

Comment:
note that anything with `noqa: I100` is has probably been ordered that way to prevent a circular import. we should disable isort on those lines.